Sources/GRPCNIOTransportHTTP2TransportServices/HTTP2ServerTransport+TransportServices.swift

```diff
@@ -264,6 +280,54 @@ extension NWProtocolTLS.Options {
         `protocol`
       )
     }
+
+    switch tlsConfig.trustRoots.wrapped {
+    case .certificates(let certificates):
+      let verifyQueue = DispatchQueue(label: "certificateVerificationQueue")
+      let verifyBlock: sec_protocol_verify_t = { (metadata, trust, verifyCompleteCallback) in
+        let actualTrust = sec_trust_copy_ref(trust).takeRetainedValue()
+
+        let customAnchors: [SecCertificate]
+        do {
+          customAnchors = try certificates.map { certificateSource in
+            let certificateBytes: Data
+            switch certificateSource.wrapped {
+            case .file(let path, .der):
+              certificateBytes = try Data(contentsOf: URL(filePath: path))
+
+            case .bytes(let bytes, .der):
+              certificateBytes = Data(bytes)
+
+            case .file(_, let format), .bytes(_, let format):
+              fatalError("Certificate format must be DER, but was \(format).")
+            }
+
+            guard let certificate = SecCertificateCreateWithData(nil, certificateBytes as CFData)
+            else {
+              fatalError("Certificate was not a valid DER-encoded X509 certificate.")
+            }
+            return certificate
+          }
+        } catch {
+          verifyCompleteCallback(false)
+          return
+        }
+
+        SecTrustSetAnchorCertificates(actualTrust, customAnchors as CFArray)
+        SecTrustEvaluateAsyncWithError(actualTrust, verifyQueue) { _, trusted, _ in
+          verifyCompleteCallback(trusted)
+        }
+      }
+
+      sec_protocol_options_set_verify_block(
+        self.securityProtocolOptions,
+        verifyBlock,
+        verifyQueue
+      )
+
+    case .systemDefault:
+      ()
```

Comment:
This is all copy-pasta from the client equivalent, can we avoid the duplication?